.gitignore

```diff
@@ -55,10 +55,6 @@ coverage.xml
 .pytest_cache/
 pytestdebug.log
 
-# Translations
-*.mo
```

Comment:
I think we should really remove compiled fixes from code and add kind of "build" pipeline with compilation.